src/postgres/interface/crc32.c

```diff
@@ -100,3 +100,24 @@ crc32cOne(const unsigned char *data, size_t size)
 
     return result ^ 0xffffffff;
 }
+
+// crc32c calculation in a few steps. this is needed to calculate the crc32c for XRecord.
+FN_EXTERN uint32_t
```

Comment:
Why not uint32 in all places?

Reply:
In pgbackrest, the variant with `_t` is used everywhere. For types without `_t` `static.vendor.h` is required.